docs/use/API_rate_limit.md

```diff
@@ -4,6 +4,42 @@
 API rate limiting is a crucial aspect of managing APIs effectively. It involves setting a threshold for the number of requests that an API can receive within a specific period. This enables you to regulate the number of incoming requests, mitigating the risk of server overload or abuse. 
 
 
+## Sessions rate limit
+
+Percona Everest API has a rate limiter to restrict the number of requests that you can make to the API within a specific time frame.  It’s the session endpoint that exchanges the user’s login and password for a JSON Web Token (JWT).
+
+By default, an Percona Everest installation allows three requests per second to this endpoint. It’s configurable via the `CREATE_SESSION_RATE_LIMIT` environment variable.
+
+
+Everest monitors failed login attempts per IP address and applies progressive timeouts to prevent unlimited login attempts without being throttled. When a rate limit is reached, the Percona Everest UI will throw an error.
+
+You can configure the session rate limit as follows:
+
+  ```sh
+  kubectl -n everest-system patch deployment percona-everest --type strategic -p 'spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    spec:
+      affinity:
+        podAntiAffinity: {}
+      containers:
+        - name: everest
+          env:
+            - name: API_REQUESTS_RATE_LIMIT
+              value: "200" 
+            - name: CREATE_SESSION_RATE_LIMIT    // <----
+              value: "3"'                        // <----
```

Comment:
There is already the same command below under `## Customize API rate limiting` section, so there is no point in having it duplicated. The only difference between this one and the existing one is that we need to add another env variable to the end of the list:  
```
            - name: CREATE_SESSION_RATE_LIMIT   
              value: "3"'   
```
Note: the `// <----` comments should be removed, in the original suggestion they were used to highlight the difference from the existing command.  

So the idea is that we can customize both API_REQUESTS_RATE_LIMIT and CREATE_SESSION_RATE_LIMIT if needed and it seems reasonable to describe both options under `## Customize API rate limiting` section

Reply:
Sure